airflow/example_dags/example_bash_operator.py

```diff
@@ -27,9 +29,9 @@
 with DAG(
     dag_id='example_bash_operator',
     schedule_interval='0 0 * * *',
-    start_date=datetime(2021, 1, 1),
+    start_date=pendulum.datetime(2021, 1, 1),
```

Comment:
```suggestion
    start_date=pendulum.datetime(2021, 1, 1, tz="UTC"),
```

Was this just missed?